src/WebsocketClient/Program.cs

```diff
@@ -0,0 +1,373 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using McMaster.Extensions.CommandLineUtils;
+using System.Net.WebSockets;
+
+namespace WebsocketClient
+{
+    class Program
+    {
+        private const int EchoMessageSize = 1000;
+
+        private static List<ClientWebSocket> _connections;
+        private static List<int> _requestsPerConnection;
+        private static List<List<double>> _latencyPerConnection;
+        private static List<(double sum, int count)> _latencyAverage;
+        private static Stopwatch _workTimer = new Stopwatch();
+        private static List<Stopwatch> _echoTimers;
+        private static bool _stopped;
+        private static SemaphoreSlim _lock = new SemaphoreSlim(1);
+        private static int _totalRequests;
```

Comment:
Is there a way to display the number of errors/bad responses, even if it's 0? That helps confirm everything went well during the run.